report/sections/Chapter 12 - Conclusion/Conclusion.tex

```diff
@@ -0,0 +1,19 @@
+The main purpose of this project was to create an internet-based application that would solve a problem. The developed system also needed to be tested. 
+In this project, we developed a system for lecturers of the \textit{Programming Paradigms} course, That would help them conduct the course. This system allowed them to define exercises with accompanying tests that students can solve. 
+The students can then receive fast feedback without the need for a teaching assistant to verify if their solution is correct. 
+
+To create this application, we had to develop a working UI that the students and lecturers could use in order to interact with the system. The UI was developed to be user-friendly and provide a minimal amount of hindrance to solving exercises.
+A backend was developed to serve the frontend to the users, and answer requests from the frontend. 
+The requests would then be rerouted to the relevant components of the system. 
```

Comment:
Please rewrite this and give more details about the Test Runner, how we use the GHC interpreter with Haskell on the backend. Mention the queue system (probably not sweeping).
Generally, more details. This is far too vague.

Reply:
Also tie it together with the suggestion above.